docker/base-images.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google LLC
```

Comment:
wondering, why don't we generate this like the rest?

Reply:
Yes, this is generated using `generate_cloudbuild.py` when run on command line.

Reply:
if the file is generated, then why are we checking it in?

Reply:
This was requested by @jonathanmetzman , sorry I'm not exactly sure of the requirement.

Reply:
I hooked up GCB to build base-images everytime FuzzBench is committed that way we never have to build base-image in CI. This speeds up CI builds quite a bit.

Reply:
Thanks! Would it make sense to explain this as a comment in the header of this file for the next person?

Reply:
Roger, I'll add the header comment.